external_review/export_DO.sh

```diff
@@ -8,6 +8,7 @@ function export_DO {
     filename="${category}_${geography_level}.csv"
     SPACES="spaces/edm-publishing/db-eddt/${category}"
     mc cp .staging/$category/$filename $SPACES/$filename
```

Comment:
Can we just export the entire `.staging/<category>` folder? That's more in line with how most of our data products work

Reply:
sure I thought the initial design was to prevent anything unintended in the final folder which I think is actually a pretty good check even though I was not the author

Reply:
I guess I just disagree. Better to create an output folder, put files in it, and send it up. I'm ok with either way for this data product I just think we usually send the entire `.output` for a reason

Reply:
Doesn't seem like I'm getting my way on this